include/pv_recorder.h

```diff
@@ -47,6 +47,7 @@ typedef enum {
  * @param frame_length The length of audio frame to get for each read call.
  * @param buffer_size_msec Time in milliseconds to store audio frames to a temporary buffer.
  * @param log_overflow Boolean variable to enable overflow logs. This will enable warning logs when buffer overflow occurs.
+ * @param log_silence Boolean variable to enable silence logs. This will log when continuous values within ABSOLUTE_SILENCE_THRESHOLD buffers are read.
```

Comment:
`ABSOLUTE_SILENCE_THRESHOLD` is an implementation detail and is also not in the header. Prob makes sense to not mention it?